chain/client/src/client.rs

```diff
@@ -669,24 +674,32 @@ impl Client {
         let block_ordinal: NumBlocks = block_merkle_tree.size() + 1;
         let prev_block_extra = self.chain.get_block_extra(&prev_hash)?;
         let prev_block = self.chain.get_block(&prev_hash)?;
-        let mut chunks = Chain::get_prev_chunk_headers(self.epoch_manager.as_ref(), &prev_block)?;
+        let (mut chunk_headers, mut chunk_endorsements) =
```

Comment:
Wait, do we actually need to copy old chunk endorsements? I think we don't need to verify copy of a chunk.
We can say that protocol behaves as follows:
- if BP includes new chunk - it must verify it by endorsements, and everyone else can verify it as well by checking signatures
- if BP doesn't - it has to include old chunk, and everyone can verify that this chunk is the same as chunk in prev block

If it makes sense, feel free to modify in following PRs.

Reply:
Discussed offline, the space savings for removing the signatures of the old chunks may be worth it. I'll include in a separate PR as I'll have to make changes to the validation logic as well.

Reply:
Created a PR: https://github.com/near/nearcore/pull/10488